packages/features/bookings/Booker/Booker.tsx

```diff
@@ -447,15 +447,11 @@ const BookerComponent = ({
         )}
       </div>
 
-      {!isPlatform ? (
-        <BookFormAsModal
-          onCancel={() => setSelectedTimeslot(null)}
-          visible={bookerState === "booking" && shouldShowFormInDialog}>
-          {EventBooker}
-        </BookFormAsModal>
-      ) : (
-        <></>
-      )}
```

Comment:
not sure to understand how this was preventing the modal to appear for web ?

Reply:
The `BookFormAsModal` is the actual component which renders the modal. Based on the above logic only if we're inside web we display the modal. As a result when we're in platform (atoms) we don't display anything, so a user won't be able to proceed further with his booking if we don't display the modal.

Reply:
gotcha